main.js

```diff
@@ -79,7 +79,22 @@ function autoUpdate () {
 }
 
 
-function ftTime(el) {
+function ftTime(dateObj) {
+	var now = new Date();
+	var interval = Math.round((now - dateObj) / 1000);
+	var dateString;
+
+	if (interval < 24 * 60 * 60 && now.getDay() === dateObj.getDay()) {
```

Comment:
Probably useful to add a comment before each if briefly mentioning what you're checking as it's something that you need to read carefully to understand and it would be much faster with a short comment